### PR TITLE
Agent: allow to configure tlsTrustCertsFilePath and useKeyStoreTls

### PR DESCRIPTION
**Pull request by eolivelli**

### CODE REVIEW

File: agent/src/main/java/com/datastax/oss/cdc/agent/AgentConfig.java [hunk 1]
```diff
@@ -249,6 +249,22 @@ public static long getEnvAsLong(String varName, long defaultValue) {
                     false, "CDC_SSL_HOSTNAME_VERIFICATION_ENABLE", Setting::getEnvAsBoolean,
                     "Boolean", "ssl", 11);
 
+    public static final String TLS_TRUST_CERTS_FILE_PATH = "tlsTrustCertsFilePath";
+    public String tlsTrustCertsFilePath;
+    public static final Setting<String> TLS_TRUST_CERTS_FILE_PATH_SETTING =
+            new Setting<>(TLS_TRUST_CERTS_FILE_PATH, Platform.ALL, (c, s) -> c.tlsTrustCertsFilePath = s, c -> c.tlsTrustCertsFilePath,
+                    "The path path to the trusted TLS certificate file.",
```
Comment (nicoloboschi) [inline agent/src/main/java/com/datastax/oss/cdc/agent/AgentConfig.java:256]: typo

File: agent/src/main/java/com/datastax/oss/cdc/agent/AgentConfig.java [hunk 2]
```diff
@@ -249,6 +249,22 @@ public static long getEnvAsLong(String varName, long defaultValue) {
...
+                    null, "CDC_TLS_TRUST_CERTS_FILE_PATH", Setting::getEnvAsString,
+                    "String", "ssl", 12);
+
+    public static final String USE_KEYSTORE_TLS = "useKeyStoreTls";
+    public boolean useKeyStoreTls;
+    public static final Setting<Boolean> USE_KEYSTORE_TLS_SETTING =
+            new Setting<>(USE_KEYSTORE_TLS, Platform.ALL, (c, s) -> c.useKeyStoreTls = Boolean.parseBoolean(s), c -> c.useKeyStoreTls,
+                    "The path path to the trusted TLS certificate file.",
```
Comment (nicoloboschi) [inline agent/src/main/java/com/datastax/oss/cdc/agent/AgentConfig.java:264]: incorrect

Comment (eolivelli) [reply agent/src/main/java/com/datastax/oss/cdc/agent/AgentConfig.java:264]: I have fixed this.
good catch !

**Comment by eolivelli** [comment]
I merged this to master branch, after fixing the typo

**Pull request closed without merging**